agent_api/src/main/java/dev/aikido/agent_api/vulnerabilities/ssrf/StoredSSRFDetector.java

```diff
@@ -21,6 +24,11 @@ public Attack run(String hostname, List<String> ipAddresses, String operation) {
             return null;
         }
 
+        ContextObject context = Context.get();
+        if (shouldSkipVulnerabilityScan(context, false)) {
```

Comment:
```suggestion
        // the 2nd param makes it so that if context is not set, we default to false.
        // this is necessary for stored SSRF where we don't want an early return even if there's no context.
        if (shouldSkipVulnerabilityScan(context, false)) {
```